### PR TITLE
Add default padding values.

### PR DESCRIPTION
**Pull request by saeta**
In the interest of progressive disclosure of complexity, and good default
values, set the padding on the layer types to be `.valid`. This follows
the Keras convention as well (https://keras.io/layers/convolutional/).

### CODE REVIEW

**Comment by rxwei** [comment]
I think you missed AvgPool.

**Comment by saeta** [comment]
Good catch. Thanks; fixed!
